io/src/ply_io.cpp

```diff
@@ -121,7 +121,13 @@ pcl::PLYReader::amendProperty (const std::string& old_name, const std::string& n
   for (; finder != cloud_->fields.rend (); ++finder)
     if (finder->name == old_name)
       break;
-  assert (finder != cloud_->fields.rend ());
+  if (finder == cloud_->fields.rend ())
+  {
+      assert (false);
+      PCL_ERROR("[pcl::PLYReader::amendProperty] old_name was not found in cloud_->fields!\n",
+          old_name.c_str());
```

Comment:
Thank you for opening this pull request! Just a small note, you forgot `%s` in the error message. And maybe move `assert (false)` after the message?

Reply:
Oh my. Done.